modules/components/org.wso2.carbon.appfactory.provisioning.runtime/src/main/java/org/wso2/carbon/appfactory/provisioning/runtime/beans/Container.java

```diff
@@ -31,6 +31,7 @@
     private String probe;
     private List<ServiceProxy> serviceProxies;
     private List<String> labels;
+    private List volumeMounts;
```

Comment:
This is fixed in following commit.
https://github.com/wso2/product-af/pull/800/commits/6d720134d55b53819ec24559da1fe0cc7b9ce447